plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveConnectorTest.java

```diff
@@ -7390,25 +7390,17 @@ public void testColumnPruning()
     }
 
     @Override
-    public void testColumnName(String columnName)
-    {
-        if (columnName.equals(" aleadingspace")) {
-            assertThatThrownBy(() -> super.testColumnName(columnName))
-                    .hasMessageMatching(format("Column name '%s' is invalid because it contains a leading space", columnName));
-            return;
-        }
-        if (columnName.equals("atrailingspace ")) {
-            assertThatThrownBy(() -> super.testColumnName(columnName))
-                    .hasMessageMatching(format("Column name '%s' is invalid because it contains a trailing space", columnName));
-            return;
+    protected boolean isColumnNameRejected(Exception exception, String columnName, boolean delimited)
+    {
+        switch (columnName) {
+            case " aleadingspace":
+                return exception.getMessage().equals("Hive column names must not start with a space: ' aleadingspace'");
```

Comment:
`exception.getMessage()` is nullable

Reply:
Good point. While it should never happen here, it's easy to flip the `.equals()`

Reply:
Yes, it should not. But say eg connector throws NPE without a message, we would get NPE from the test, instead of actual query failure (a different NPE).